### DISCUSSION TITLE
Windows Enterprise 20H2?

### DISCUSSION THREAD

**Discussion by kmajeed** [Fs00/Win10BloatRemover#1] [Questions] [answered]
Does the latest version support Windows Enterprise 20H2?

**Comment by Fs00** [comment, marked as answer]
Hi @kmajeed! The next version with 20H2 support is in the works and will hopefully be released before the end of the year (or in January at worst). The current one (4.0) supports only 2004 and 1909.
The program should work without issues on Enterprise edition. I test it on Windows Education among others, which behaves very similarly to Enterprise if I'm not mistaken.
